specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/examples/ServerEndpoints_Create.json

```diff
@@ -12,8 +12,8 @@
                 "cloudTiering": "off",
                 "volumeFreeSpacePercent": 100,
                 "tierFilesOlderThanDays": 0,
-                "cloudSeededData": "on",
-                "cloudSeededDataFileShareUri" : "https://storageaccountname.file.core.windows.net/filesharename"
+                "offlineDataTransfer": "on",
+                "offlineDataTransferShareName": "myfileshare"
```

Comment:
Looking at the model, these two properties are nested properties of a property with name `parameters` https://github.com/Azure/azure-rest-api-specs/blob/73b14d60d8d686e0626fb6cdb67c41e0a90b9023/specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json#L1466